cityofaustin/formstack-alpha-theme
# 3775 Save and resume confirmation page is missing

Zenhub issue: https://app.zenhub.com/workspaces/techstack-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/3775

So, when a resident wants to fill out one of our forms... 
- **but** needs to save their form to continue later,
- the "Save and Resume Later" is used to email a link to the current state of their form. 
- Cool, but their's no confirmation message. 

### This fixes that ☝️

After me and Nick lost our minds together. We discovered a very particular thing happening with jQuery and some infighting with Prototypes.
- In short, we can't use "$" for referencing jQuery when our custom code is being used on PHP forms! eek!

For more info, see comments in code...
- Only file actually modified outside of setting up test data was `src/footer.hbs`.

## To test 

Visit test url: https://cityofaustin.formstack.com/forms/bob_s_simple_form
- Click [save and resume later]
- enter email `ignore@igore.com`
- did you see a confirmation message - small as it may be?
- NOTE: this definitely needs to be styles ;) -> font is super small.

### Review and QA Passed. 
- Copy/paste `theme/header.html` and `theme/footer.html` into formstack's text box in advanced code editor.
- Oh, and test OPO forms as well!
